### DISCUSSION TITLE
How to migrate from a LinuxServer.io docker image to AIO?

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#1543] [Questions] [answered]
How to migrate from a LinuxServer.io docker image to AIO? The instructions in https://github.com/nextcloud/all-in-one/blob/main/migration.md doesn't list a existing docker installation.

**Comment by szaimen** [comment, marked as answer]
Hi, the documentation should work for docker images as well or what exactly are you missing/what did not work for you?

**Reply by apoorv569** [reply]
> Hi, the documentation should work for docker images as well or what exactly are you missing/what did not work for you?

Sorry, I have already resolved this. I only have 3-4 users so I just deleted the old instance and copied the users data to another place, installed AIO and copied the data back for users.

How can I close this discussion?

**Reply by szaimen** [reply]
> How can I close this discussion?

I'll mark it as resolved for you 👍
